docs/Installation/Installation.md

```diff
@@ -19,7 +19,7 @@ installation_on_clusters "Installation on clusters" page.
 * [Git](https://git-scm.com/)
 * BLAS (e.g. [OpenBLAS](http://www.openblas.net))
 * [Blaze](https://bitbucket.org/blaze-lib/blaze/overview) v3.5
-* [Boost](http://www.boost.org/) 1.60.0 or later
+* [Boost](http://www.boost.org/) between 1.63.0 and 1.72.0
```

Comment:
Please don't make unrelated changes in your PRs.  This is only correct for OSX (or generally systems with case-insensitive file systems).  Higher versions work fine in other cases.

Reply:
Sure, I dropped this commit for now and will move it to a separate PR.